### PR TITLE
(maint) - 2020-07-31 IAC Blog Post

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: _posts/2020-07-31-status-update.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Community Contributions
+We've had plenty of great community contributions this week:
+- 
+
+## Newly released blog
+In responce to some questions from the community the wonderfuly talented [David Swan][david_swan] a proud member of the IAC team has released a new blog post detailing the use of Litmus with Vagrant, covering both the basic usage as well as some more detailed functions. So if you have some free time or just want to learn more about Vagrant please give it a look. You can find it through the link shown [here][% link _docs/LitmusVagrantTesting.md %}].
+
+## Puppet 7
+Testing with the Puppet 7 nightly builds began in earnest, this week.
+Whilst building out our automation for more extensive and regular testing (_using [pdksync]({% link tools/pdksync.html %}) at it's core_), we have been running tests against a subset of the [supported modules]({% link modules.html %}).
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:36]:
```suggestion
Whilst building out our automation for more extensive and regular testing (_using [pdksync][pdksync] at it's core_), we have been running tests against a subset of the [supported modules]({% link modules.html %}).
```

File: _posts/2020-07-31-status-update.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+[websphere_application_server-forge]:       https://forge.puppet.com/puppetlabs/websphere_application_server
+[websphere_application_server-2-1-0]:       https://forge.puppet.com/puppetlabs/websphere_application_server/2.1.0/changelog
+[puppetlabs-firewall]:                      https://github.com/puppetlabs/puppetlabs-firewall
+[firewall-forge]:                           https://forge.puppet.com/puppetlabs/firewall
+[firewall-2-5-0]:                           https://forge.puppet.com/puppetlabs/firewall/2.5.0/changelog
+[david_swan]:                               https://github.com/david22swan
+[mihaibuzgau]:                              https://github.com/mihaibuzgau
+[gimmyxd]:                                  https://github.com/gimmyxd
+[joshcooper]:                               https://github.com/joshcooper
+[binford2k]:                                https://github.com/binford2k
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:57]:
```suggestion
[binford2k]:                                https://github.com/binford2k
[pdksync]:                                  https://puppetlabs.github.io/iac/tools/pdksync.html
```

File: _posts/2020-07-31-status-update.md (hunk 1 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md]: Done

File: _posts/2020-07-31-status-update.md (hunk 2 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md:69]: Done

File: _posts/2020-07-31-status-update.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Module Releases from IAC Team
+We've had a few modules released this week:
+- [puppetlabs-firewall][firewall-forge][`v2.5.0`][firewall-2-5-0]
+- [puppetlabs-websphere_application_server][websphere_application_server-forge][`v2.1.0`][websphere_application_server-2-1-0]
+
+## Community Contributions
+We've had plenty of great community contributions this week:
+- [kenyon][kenyon] [spotted a mistake in the supported list][docker-641] for the [Docker module][docker].
+- [kajinamit][kajinamit] has given us two prs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second set's the [python3-mod_wsgi to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md]:
```suggestion
- [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second sets the [python3-mod_wsgi MOD to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.
```

File: _posts/2020-07-31-status-update.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- [bFekete][bFekete] has put in a pr to [increase the puppetlabs-concat lower bound][tomcat-404] within the [Tomcat module][tomcat].
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md]:
```suggestion
- [bFekete][bFekete] has put in a PR to [increase the puppetlabs-concat lower bound][tomcat-404] within the [Tomcat module][tomcat].
```

File: _posts/2020-07-31-status-update.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+## Newly released blog
+In responce to some questions from the community the wonderfuly talented [David Swan][david_swan] a proud member of the IAC team has released a new blog post detailing the use of Litmus with Vagrant, covering both the basic usage as well as some more detailed functions. So if you have some free time or just want to learn more about Vagrant please give it a look. You can find it through the link shown [here][litmus-vagrant].
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:34]:
```suggestion
In response to some questions from the community, the wonderfully talented [David Swan][david_swan], a proud member of the IAC team, has released a new blog post detailing the use of Litmus with Vagrant, covering both the basic usage as well as some more detailed functions. So if you have some free time or just want to learn more about Vagrant please give it a look. You can find it through the link shown [here][litmus-vagrant].
```

File: _posts/2020-07-31-status-update.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+## Puppet 7
+Testing with the Puppet 7 nightly builds began in earnest, this week.
+Whilst building out our automation for more extensive and regular testing (_using [pdksync](pdksync) at it's core_), we have been running tests against a subset of the [supported modules]({% link modules.html %}).
+So far, testing has generally been quite positive, with some minor issues arising.
+Fortunately, we work with awesome colleagues who are very quick to react to any issues we discover, helping with the debugging process and providing fixes really quickly.
+Thanks to [joshcooper][joshcooper], we hope to have a fix for one of these issues quite soon!
+[gimmyxd][gimmyxd] and [mihaibuzgau][mihaibuzgau] are helping to analyse the other issue we're currently encountering and [binford2k][binford2k] has been assisting in how we communicate any potential impacts to the Puppet Community.
+On that note, it goes without saying, we are doing everything possible to make the jump to Puppet 7 as seamless as possible and provide clear communication to the Community for anything that you will need to be aware of.
+
+## DSC Revamp
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:45]: Is this a placeholder or should it be removed?

File: _posts/2020-07-31-status-update.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+## CI update
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:47]: Is this a placeholder or should it be removed?

File: _posts/2020-07-31-status-update.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+## Module Releases from IAC Team
+We've had a few modules released this week:
+- [puppetlabs-firewall][firewall-forge][`v2.5.0`][firewall-2-5-0]
+- [puppetlabs-websphere_application_server][websphere_application_server-forge][`v2.1.0`][websphere_application_server-2-1-0]
+
+## Community Contributions
+We've had plenty of great community contributions this week:
+- [kenyon][kenyon] [spotted a mistake in the supported list][docker-641] for the [Docker module][docker].
+- [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second set's the [python3-mod_wsgi to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:30]:
```suggestion
- [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second sets the [MOD package to `python3-mod_wsgi` in place of `mod_wsgi` on CentOS 8][apache-2052] as the older name no longer works.
```

File: _posts/2020-07-31-status-update.md (hunk 3 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md]: Updated to:
- [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second sets [python3-mod_wsgi to be used in place of mod_wsgi on CentOS 8][apache-2052] as the older name no longer works.

File: _posts/2020-07-31-status-update.md (hunk 4 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md]: Done

File: _posts/2020-07-31-status-update.md (hunk 5 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md:34]: Done

File: _posts/2020-07-31-status-update.md (hunk 6 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md:45]: Was a placeholder

File: _posts/2020-07-31-status-update.md (hunk 7 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md:47]: Removed

File: _posts/2020-07-31-status-update.md (hunk 8 above)
Comment (david22swan) [reply _posts/2020-07-31-status-update.md:30]: - [kajinamit][kajinamit] has given us two PRs for the [Apache module][apache], the first of which [standardises the use of distrelease within the module][apache-2053] to ensure that the same logic is used to retrieve the Major version throughout, while the second sets [`python3-mod_wsgi` to be used in place of `mod_wsgi` on CentOS 8][apache-2052] as the older name no longer works.

File: _posts/2020-07-31-status-update.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  - status
+tags:
+  - releases
+  - pdk
+  - shoutouts
+  - security
+  - rspec-puppet
+  - java
+  - inifile
+  - docker
```
Comment (sanfrancrisko) [inline _posts/2020-07-31-status-update.md:16]:
```suggestion
  - docker
  - puppet7
```

Comment (sanfrancrisko) [reply _posts/2020-07-31-status-update.md:16]: Oh...and should `rspec-puppet` and `pdk` be in there?

Comment (sanfrancrisko) [reply _posts/2020-07-31-status-update.md:16]: ...and `java`, `inifile`? Looks like last week's tags

Comment (david22swan) [reply _posts/2020-07-31-status-update.md:16]: Updated tags

**Pull request merged**